conf/messages.en

```diff
@@ -315,6 +317,21 @@ unloadingFindings.rowHeadings.additionalComments = Comments
 unloadingFindings.rowHeadings.additionalComments.change.hidden = comments
 unloadingFindings.rowHeadings.additionalComments.remove.hidden = comments
 
+checkYourAnswers.rowHeadings.goodsUnloadedDate = Goods’ unloaded date
+checkYourAnswers.rowHeadings.goodsUnloadedDate.change.hidden = goods’ unloaded date
+checkYourAnswers.rowHeadings.canSealsBeRead = Are all of the seal identification numbers or marks readable?
+checkYourAnswers.rowHeadings.canSealsBeRead.change.hidden = are all of the seal identification numbers or marks readable?
```

Comment:
Is there content for this? I thought it'd have been `if all of the seal identification numbers or marks are readable` so the a11y text will read `Change if all of the seal identification numbers or marks are readable`